rohmu/object_storage/s3.py

```diff
@@ -322,7 +322,7 @@ def _get_object_stream(self, key: str, byte_range: Optional[tuple[int, int]]) ->
         path = self.format_key_for_backend(key, remove_slash_prefix=True)
         kwargs: dict[str, Any] = {}
         if byte_range:
-            kwargs["Range"] = f"bytes {byte_range[0]}-{byte_range[1]}"
+            kwargs["Range"] = f"bytes={byte_range[0]}-{byte_range[1]}"
```

Comment:
From here: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/get_object.html There is also an example:
```
response = client.get_object(
    Bucket='examplebucket',
    Key='SampleFile.txt',
    Range='bytes=0-9',
)

print(response)
```

I guess Markus got confused because the *output* uses space? 
```
{
    'AcceptRanges': 'bytes',
    'ContentLength': '10',
    'ContentRange': 'bytes 0-9/43',
    'ContentType': 'text/plain',
    'ETag': '"0d94420ffd0bc68cd3d152506b97a9cc"',
    'LastModified': datetime(2014, 10, 9, 22, 57, 28, 3, 282, 0),
    'Metadata': {
    },
    'VersionId': 'null',
    'ResponseMetadata': {
        '...': '...',
    },
}
```